src/main/java/com/udelphi/repository/RoleRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.udelphi.repository;
+
+import com.udelphi.model.Role;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+import java.util.List;
+import java.util.Set;
+
+public interface RoleRepository extends JpaRepository<Role, Integer> {
+    @Query("select u.roles from User u where u.id =:id")
```

Comment:
do we really need query?

Reply:
Sping Data can`t automatically generate query because the role hasn`t list users